### PR TITLE
Create endpoint to get confirm goods delivery for recipient/sender

### PR DESCRIPTION
**Pull request by tonia4ik-s**

### CODE REVIEW

File: Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        public float GoodsWeight { get; set; }
+        public string Address { get; set; }
+        public string Settlement { get; set; }
+        public string Region { get; set; }
+        public string DriversName { get; set; }
+        public string DriversSurname { get; set; }
+        public string CarModel { get; set; }
+        public string CarNumber { get; set; }
+    }
+}
```
Comment (Grasmus) [inline Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs]: Add empty line in the eof

File: API/Controllers/OfferController.cs [hunk 2]
```diff
@@ -83,5 +83,13 @@ public async Task<IActionResult> DeleteAsync([FromQuery] OfferIdDTO offerIdDTO)
 
             return Ok();
         }
+        
+        [HttpGet("to-confirm")]
+        [AuthorizeByRole(IdentityRoleNames.User)]
+        public async Task<IActionResult> GetUserOffersToConfirmAsync([FromQuery] PaginationFilterDTO paginationFilter)
+        {
+            var userId = _userService.GetCurrentUserNameIdentifier(User);
```
Comment (Grasmus) [inline API/Controllers/OfferController.cs:92]: Add empty line

File: Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Core.DTO.OfferDTO
+{
+    public class OfferPreviewForConfirmDTO
```
Comment (Grasmus) [inline Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs:5]: How will we know if this offer is confirmed or not?

File: Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        public string Description { get; set; }
+        public bool IsClosed { get; set; }
+        public bool IsAnsweredByCreator { get; set; }
+        public bool IsAnsweredByDriver { get; set; }
+        public float GoodsWeight { get; set; }
+        public string Address { get; set; }
+        public string Settlement { get; set; }
+        public string Region { get; set; }
+        public string DriversName { get; set; }
+        public string DriversSurname { get; set; }
```
Comment (Grasmus) [inline Core/DTO/OfferDTO/OfferPreviewForConfirmDTO.cs]: We already have DTO for this fields

**Review by Grasmus — APPROVED** [review]

**Review by AndriiChepeliuk — APPROVED** [review]

**Pull request merged**